platform-sdk/swirlds-common/src/main/java/module-info.java

```diff
@@ -181,4 +181,5 @@
     /* Prometheus Java client */
     requires io.prometheus.simpleclient;
     requires io.prometheus.simpleclient.httpserver;
+    requires static com.github.spotbugs.annotations;
```

Comment:
are we starting to use these annotations throughout our code?

Reply:
yes. it is defined in the code format spec